src/webapp/forms.py

```diff
@@ -1,9 +1,9 @@
 import datetime
 
 from flask_wtf import FlaskForm
-from wtforms import SelectField, SelectMultipleField, StringField, \
-    TimeField, TextAreaField, SubmitField
-from wtforms.fields.html5 import DateTimeLocalField
+from wtforms.fields import SelectField, SelectMultipleField, StringField, \
```

Comment:
Is there any difference between the imports from `wtforms` and `wtforms.fields` ?

Reply:
No, this can be changed back. Looking at it again the previous way is more inline with the documentation.

Reply:
https://wtforms.readthedocs.io/en/2.3.x/crash_course/#exploring-in-the-console